packages/admin/admin-color-picker/src/ColorPicker.tsx

```diff
@@ -120,6 +123,14 @@ const ColorPicker = ({
                     </InputAdornment>
                 )
             }
+            endAdornment={
```

Comment:
What happens if there's no `clearable` and no `endAdornment`? The input's `endAdornment` would be `<></>`, which is not `undefined`. Does MUI apply some styling for non `undefined` endAdornments? 🤔

Reply:
Good point! 

There's no change in our components that I could find, but when using out-of-the-box Mui components there is a change in the label behavior when using `startAdornment` with `<></>`. I'll change this just to be safe, maybe this would cause an issue in the future.